unfurl/server.py

```diff
@@ -139,6 +142,78 @@ def export():
 
     return jsonify(json_summary)
 
+
+@app.route("/update_deployment", methods=["POST"])
+def update_deployment():
+    body = request.json
+    # Repository URL
+    project_path = body.get("projectPath")
+    # File path
+    path = body.get("path")
+    # Patch
+    patch = body.get("patch")
+
+    # Project is external
+    if project_path.startswith('http') or project_path.startswith('git'):
+        clone_root = current_app.config["UNFURL_CLONE_ROOT"]
```

Comment:
We don't want to clone a new repo for every patch request! So let's check for the existence for the repo just like you do in the export endpoint (using find_git_repo())